starport/pkg/xrelayer/relayer.go

```diff
@@ -23,7 +23,7 @@ func Link(ctx context.Context, paths ...string) (linkedPaths, alreadyLinkedPaths
 // Start relays tx packets for paths until ctx is canceled.
 func Start(ctx context.Context, paths ...string) error {
 	var reply interface{}
-	return tsrelayer.Call(ctx, "start", paths, &reply)
+	return tsrelayer.Call(ctx, "start", []interface{}{paths}, &reply)
```

Comment:
I think it is a good pattern to always wrap the request inside an array to create a standard on this. Otherwise we mix both styles which is hard to keep track.

Reply:
ok, fair enough